packages/backend/src/services/vendor/driver/companyHouseUkDriver.ts

```diff
@@ -60,18 +54,109 @@ export class CompanyHouseUkDriver extends BaseDetailsSearchDriver<
   }
 
   /**
-   * https://developer-specs.company-information.service.gov.uk/companies-house-public-data-api/reference/company-profile/company-profile
+   * GET /company/{companyNumber}
+   * see https://developer-specs.company-information.service.gov.uk/companies-house-public-data-api/reference/company-profile/company-profile
    */
   async getDetails(
     integration: VendorIntegration,
     detailsOptions: DetailsOptions
   ): Promise<VendorResult<CompanyHouseUkDetailsResponse>> {
+    const url = new URL(`https://${CH_DOMAIN}/company/${detailsOptions.searchResultId}`);
+    const result = await this.get<DetailsResponseBody>(integration, url);
+    const properties = flattenJson(result) as CompanyHouseUkDetailsResponse;
+    this.fixLinks(properties);
+
+    // load optional neighbors
+    const neighbors: NeighborResult[] = [];
+    if (integration.getAdminSettings('officers') === 'Yes') {
```

Comment:
it's not just a boolean?

Reply:
ok, boolean were not available initially for admi nsettings, but let's implement them